### PR TITLE
GraphQL/PeeWee: rename 'items' to 'numberOfItems' and 'number_of_items'

### PR DESCRIPTION
**Pull request by spaudanjo**
Closes https://trello.com/c/sano156L/938-new-app-use-consistently-numberofitems-numberofitems-in-python-instead-of-items

### CODE REVIEW

**Comment by spaudanjo** [comment]
@pylipp @aerinsol I'm actually wondering whether `quantity` is not even a better name (plus, one double-naming candidate less - we can call it both in the GraphQL and the Python internal worlds the same since it has just one word - instead of numberOfItems which would also have to be written - for Python - number_of_items). 
If you both agree, I can change it quickly. 

Also be aware that this doesn't change the database column names of course - so no changes for Dropapp. 
It would require query adaptions though (very easy/quick ones) for any partner using the API though.

**Comment by aerinsol** [comment]
@spaudanjo I am ok with changing it, but I would like _item_ to be part of the naming, as _quantity_ is vague (boxes? Something else?). Not worried about leaving the db named the same, that's one of the benefits of using an ORM - no costly schema migrations.

I will leave @pylipp to comment on the other points.

File: back/boxtribute_server/graph_ql/resolvers.py [hunk 1]
```diff
@@ -146,11 +146,6 @@ def resolve_packing_list_entry_matching_packed_items_collections(obj, *_):
     )
     unboxed_items_colletions = UnboxedItemsCollection.select(
         UnboxedItemsCollection,
```
Comment (pylipp) [inline back/boxtribute_server/graph_ql/resolvers.py:148]: For `select *` you can omit the part in the parentheses.

File: back/boxtribute_server/graph_ql/types.graphql [hunk 2]
```diff
@@ -32,7 +32,7 @@ type Box implements ItemsCollection {
   place: BoxPlace
   distributionEvent: DistributionEvent
   " The number of items the box contains. "
```
Comment (pylipp) [inline back/boxtribute_server/graph_ql/types.graphql:34]: you can remove this comment now, the field name is self-explanatory

File: react/src/views/Boxes/BoxesView.test.tsx [hunk 3]
```diff
@@ -34,7 +34,7 @@ describe("Boxes view", () => {
                         gender: "Women",
                         name: "Long Dress",
                       },
-                      items: 64,
+                      itnumberOfItemsems: 64,
```
Comment (pylipp) [inline react/src/views/Boxes/BoxesView.test.tsx:37]: This doesnt look right :laughing:

File: react/src/views/Boxes/BoxesView.test.tsx [hunk 4]
```diff
@@ -126,7 +126,7 @@ describe("Boxes view", () => {
                         gender: "Boy",
                         name: "Top Boys (18-24 months)",
                       },
-                      items: 95,
+                      itnumberOfItemsems: 95,
```
Comment (pylipp) [inline react/src/views/Boxes/BoxesView.test.tsx:129]: same

**Review by pylipp — CHANGES_REQUESTED** [review]
Search&replace FTW, minor things to clean up :smiley:

**Comment by pylipp** [comment]
[New App: use consistently numberOfItems (number_of_items in Python) instead of items](https://trello.com/c/sano156L/938-new-app-use-consistently-numberofitems-numberofitems-in-python-instead-of-items)

File: react/src/views/Boxes/BoxesView.test.tsx (hunk 3 above)
Comment (spaudanjo) [reply react/src/views/Boxes/BoxesView.test.tsx:37]: You passed the test! #weeklyPrReviewQualityAssurance

**Comment by spaudanjo** [comment]
addressed you comments @pylipp 🌞

**Review by pylipp — APPROVED** [review]

**Pull request merged**